src/components/PeopleFilters.tsx

```diff
@@ -1,18 +1,72 @@
+import React from 'react';
+import { Link, useLocation, useSearchParams } from 'react-router-dom';
+import { getSearchWith, SearchParams } from '../utils/searchHelper';
+
 export const PeopleFilters = () => {
+  const availableCenturies = [16, 17, 18, 19, 20];
```

Comment:
```suggestion
  const availableCenturies = [16, 17, 18, 19, 20];
```

We can move this variable to outside of the component, since we aren't changing it's values anywhere in the code flow, so we can avoid it's re-creation across re-renders =)